.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: envattrs release workflow
+
+on:
+  push:
+    tags:
+      - '*'
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: Release to PyPI
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.6
+      - run: |
+          curl -sSL https://install.python-poetry.org | python - \
+        env:
+          POETRY_VERSION: 1.1.15
+        shell: bash
+      - name: Release
+        run: poetry publish --build
```

Comment:
Sorry if I missed it, but where do you set the pypi URL to release this package?

Reply:
You mean the production pypi url is it?
The publish command will by default publish to the production pypi.

Reply:
my take: it's not necessary to inject pypi url.

If the publication setup was more complex, then being able to test against test.pypy.org could be a reasonable ask. As it stands now, I don't feel that's needed.